core/bounds.gms

```diff
@@ -364,13 +364,13 @@ vm_capEarlyReti.fx(ttot,regi,"dot")=0;
 vm_deltaCap.up(t,regi,"dot","1")$( (t.val gt 2005) AND regi_group("EUR_regi",regi) )  = 1e-6;
 
 
-
+*' @code{extrapage: "00_model_assumptions"}
 *** -----------------------------------------------------------------------------
-*DK 20100929 Bound on CCS injection rate
+*' #### Bound on maximum annual carbon storage by region
 *** -----------------------------------------------------------------------------
-*** default value (0.5%) is consistent with Interview Gerling (BGR)
-*** http://www.iz-klima.de/aktuelles/archiv/news-2010/mai/news-05052010-2/
-*** 12 Gt storage potential in Germany, 50-75 Mt/a injection => 60 Mt/a => 60/12000=0.005
+*' DK 20100929: default value (pm_ccsinjecrate= 0.5%) is consistent with Interview Gerling (BGR)
+*' (http://www.iz-klima.de/aktuelles/archiv/news-2010/mai/news-05052010-2/): 
+*' 12 Gt storage potential in Germany, 50-75 Mt/a injection => 60 Mt/a => 60/12000=0.005
```

Comment:
This explanation is outdated and the link doesn't work anymore. Now with REMIND EU we have very different assumptions for Germany, namely 110 MtCO2/yr instead of 60. I assume, EU numbers for total storage potential might be downscaled or the total storage numbers have been updated since 2010. We should double check and adapt the comment as a CarMa group effort.